receiver/kubeletstatsreceiver/scraper.go

```diff
@@ -67,8 +82,21 @@ func newKubletScraper(
 			metricsConfig.Metrics.K8sPodMemoryRequestUtilization.Enabled ||
 			metricsConfig.Metrics.K8sContainerMemoryLimitUtilization.Enabled ||
 			metricsConfig.Metrics.K8sContainerMemoryRequestUtilization.Enabled,
+		stopCh:   make(chan struct{}),
+		nodes:    make(map[string]kubelet.NodeLimits),
```

Comment:
Why do we need to maintain this map if we need only one node?

Reply:
Good point. This map might became redundant after some point. We can do with a single struct since we only focus and watch on one specific node. Thank's for spotting this!

changed: [39ec4b3](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/32295/commits/39ec4b3f97a3165324cc28f975d7125c62987ff9)